### DISCUSSION TITLE
Desktop version

### DISCUSSION THREAD

**Discussion by JonPlayGo** [adrcotfas/goodtime#324] [General]
Is a desktop version of the application expected?(Linux,Windows)

**Comment by adrcotfas** [comment]
I'm currently rewriting the app in KMP so it will be possible without much effort in future releases. But first I'll focus on the Android and iOS versions.

**Reply by HerrPischinger** [reply]
Any updates on a possible Desktop Version? Love the app on android

**Reply by adrcotfas** [reply]
It will come at some point in the future after the iOS version.
